src/systemctl/systemctl.c

```diff
@@ -143,6 +144,7 @@ static const char *arg_kill_who = NULL;
 static int arg_signal = SIGTERM;
 static char *arg_root = NULL;
 static usec_t arg_when = 0;
+static char *arg_esp_path;
```

Comment:
this really should be initialized to NULL. While on Linux is it safe not to do this, valgrind does notice, and ANSI C isn't fine with this either. Also, I think it's more readable if we initialize global variables explicitly, and we do that for all others too